pipelines/deploy-pipelines.yml

```diff
@@ -249,7 +258,30 @@ jobs:
             grafana/k6:latest \
             run /app/test/performance/idpayOnboardingAPI.js
         displayName: Run k6 idpayOnboardingAPI.js
-        condition: in('${{parameters.SCRIPT }}', 'idpayOnboardingAPI', 'ioPutEnrollInstrumentIssuer', 'ioPutEnrollIban', 'rtdTransactionsFile') 
+        condition: in('${{parameters.SCRIPT }}', 'idpayOnboardingAPI', 'idPayPaymentDiscountAPI', 'ioPutEnrollInstrumentIssuer', 'ioPutEnrollIban', 'rtdTransactionsFile')
+        timeoutInMinutes: 0
+      - script: |
+          docker run -i \
+            --user $UID \
+            -v $(pwd):/app \
+            --workdir "/app" \
+            -e RESULTS_DIR=/app \
+            -e TARGET_ENV=${{ parameters.TARGET_ENV }} \
+            -e SCENARIO_TYPE_ENV=${{ parameters.SCENARIO_TYPE_ENV }} \
+            -e VIRTUAL_USERS_ENV=${{ parameters.VIRTUAL_USERS_ENV }} \
+            -e STAGE_NUMBER_ENV=${{ parameters.STAGE_NUMBER_ENV }} \
+            -e VUS_MAX_ENV=${{ parameters.VUS_MAX_ENV }} \
+            -e START_TIME_ENV=${{ parameters.START_TIME_ENV }} \
+            -e DURATION_PER_VU_ITERATION=${{ parameters.DURATION_PER_VU_ITERATION }} \
+            -e RATE=${{ parameters.RATE }} \
+            -e ONE_SCENARIO=${{ parameters.ONE_SCENARIO }} \
+            -e APIM_SK=$(acquirerApiKey) \
```

Comment:
On pipeline execution logs I see the following message: "Tracing authorization for this subscription has expired on". Are you using the right API key?